client_options.go

```diff
@@ -118,6 +118,14 @@ func WithResponseLogHook(hook retryablehttp.ResponseLogHook) ClientOptionFunc {
 	}
 }
 
+// WithErrorHandler can be used to configure a custom error handler.
+func WithErrorHandler(handler retryablehttp.ErrorHandler) ClientOptionFunc {
+	return func(c *Client) error {
+		c.client.ErrorHandler = handler
+		return nil
+	}
+}
```

Comment:
These options are ordered alphabetically, so if you can update the location of the new option that would be great...

Reply:
Thanks for the quick response! Fixed this :)